README.md

```diff
@@ -659,6 +659,19 @@ terraform state rm aws_elasticsearch_domain.elasticsearch
 … to update the Terraform state so that it reflects the deletion of the
 Elasticsearch domain. Now running `make deploy` should succeed. 
 
+
+## Terraform fails to update `tfstate` while running `make deploy`
+Terraform can successfully run by creating, refreshing and deleting resources 
```

Comment:
>is that statement there to inform us of a successful terraform run looks like?

Yea, but I simply included the traceback of the error to avoid all the other complexities.